tests/unit/test-class-wc-payments-express-checkout-button-display-handler.php

```diff
@@ -302,25 +270,4 @@ public function test_display_express_checkout_buttons_only_woopay() {
 		$this->assertStringNotContainsString( 'wcpay-express-checkout-button-separator', ob_get_contents() );
 		ob_end_clean();
 	}
-
-	public function test_display_express_checkout_buttons_only_payment_request() {
-		$this->mock_woopay_button_handler
-			->method( 'should_show_woopay_button' )
-			->willReturn( false );
-
-		$this->mock_payment_request_button_handler
-			->method( 'should_show_payment_request_button' )
-			->willReturn( true );
-
-		$this->mock_express_checkout_helper
-			->method( 'is_checkout' )
-			->willReturn( true );
-
-		ob_start();
-		$this->express_checkout_button_display_handler->display_express_checkout_buttons();
-
-		$this->assertStringContainsString( 'wcpay-express-checkout-button-separator', ob_get_contents() );
-		$this->assertStringContainsString( 'display:none;', ob_get_contents() );
-		ob_end_clean();
-	}
```

Comment:
This whole test case is no longer needed because the `payment_request_button_handler` has been removed.